### PR TITLE
Bump the prod env to v3.2.1

### PR DESCRIPTION
**Pull request by LinuxSuRen**
The INT env has already upgraded to v3.2.1 (https://github.com/kubesphere-sigs/ks-infra/pull/5) successfully. See the following screenshot:

image

/cc @kubesphere-sigs/sig-devops

### CODE REVIEW

File: prod/kubesphere/kubesphere-installer.yaml [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ks-installer
+  namespace: kubesphere-system
+  labels:
+    app: ks-install
+    version: v3.2.0
```
Comment (JohnNiang) [inline prod/kubesphere/kubesphere-installer.yaml:277]: Why is this version `v3.2.0`? Please see https://github.com/kubesphere/ks-installer/blob/73e20baddf715f8b16027a6fca4b419c12590e79/deploy/kubesphere-installer.yaml.

File: prod/kubesphere/kubesphere-installer.yaml [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+  annotations:
+    deployment.kubernetes.io/revision: '1'
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: ks-install
+      version: v3.2.0
```
Comment (JohnNiang) [inline prod/kubesphere/kubesphere-installer.yaml:285]: See above.

File: prod/kubesphere/kubesphere-installer.yaml [hunk 3]
```diff
@@ -0,0 +1,320 @@
...
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: ks-install
+        version: v3.2.0
```
Comment (JohnNiang) [inline prod/kubesphere/kubesphere-installer.yaml:291]: See above.

File: prod/kubesphere/kubesphere-installer.yaml (hunk 1 above)
Comment (LinuxSuRen) [reply prod/kubesphere/kubesphere-installer.yaml:277]: I guess `version: v3.2.0` was generated. And k8s does not allow to modify the label selector.

Comment (LinuxSuRen) [reply prod/kubesphere/kubesphere-installer.yaml:277]: How is it going?

**Review by JohnNiang — APPROVED** [review]
/lgtm

**Pull request merged**